scripts/split-e2e-tests-v2.ts

```diff
@@ -104,6 +104,8 @@ const TEST_EXCLUSIONS: { l: string[]; w: string[] } = {
     'src/__tests__/opensearch-simulator/opensearch-simulator.test.ts',
     'src/__tests__/storage-simulator/S3server.test.ts',
     'src/__tests__/amplify-app.test.ts',
+    // failing in parsing JSON strings on powershell
+    'src/__tests__/auth_12.test.ts',
```

Comment:
Can you add work item in ops backlog to find a solution for these problems JSON parameters and Powershell?
It's not the only tests that have these problems.